main.js

```diff
@@ -37,6 +39,30 @@ if (pathToParamedicConfig || // --config
         ParamedicConfig.parseFromFile(pathToParamedicConfig):
         ParamedicConfig.parseFromArguments(argv);
 
+    if(argv.plugin) {
+        paramedicConfig.setPlugins(argv.plugin);
+    }
+
+    if(argv.outputDir) {
+        paramedicConfig.setOutputDir(argv.outputDir);
+    }
+
+    if(argv.logMins) {
+        paramedicConfig.setLogMins(argv.logMins);
+    }
+
+    if(argv.tccDbPath){
```

Comment:
Do you assume the tccDbPath exists? Perhaps you validate somewhere else - should you instead validate here?